internal/db/schema/migrations/oss/postgres/76/01_deleted_tables_and_triggers.up.sql

```diff
@@ -0,0 +1,247 @@
+-- Copyright (c) HashiCorp, Inc.
+-- SPDX-License-Identifier: MPL-2.0
+
+begin;
+  create table target_deleted (
+    public_id wt_public_id primary key,
+    delete_time wt_timestamp
+  );
+
+  create index target_deleted_time_idx on target_deleted (delete_time);
```

Comment:
I think this should be
```suggestion
  create index target_deleted_delete_time_idx on target_deleted (delete_time);
```
to be consistent with the recommendation for unique constraints in [the SQL style guide](https://github.com/hashicorp/boundary/blob/main/sql-style-guide.md).